README.md

```diff
@@ -18,7 +18,9 @@
   Join our active, engaged community: <br>
   <a href="https://saleor.io/">Website</a>
   <span> | </span>
-  <a href="https://medium.com/saleor">Blog</a>
+  <a href="https://saleor.io/blog/">Blog</a>
+  <span> | </span>
+  <a href="https://saleor.typeform.com/to/JTJK0Nou">Newsletter</a>
```

Comment:
Shouldn't be something like `Subscribe to newsletter`? Personally, I would think that this link will redirect me to a page with all news, but maybe I am wrong 😄

Reply:
I agree, it would be more clear that way 👍

Reply:
Done